src/table/head/table-head.component.ts

```diff
@@ -86,6 +88,8 @@ export class TableHead {
 
 	@Input() stickyHeader = false;
 
+	@Input() sortable = true;
```

Comment:
Maybe add a comment to note that disabling this will disable sorting **for all headers, including sortable ones** and to generally prefer setting the sortable state on the header item.